lib/action_subscriber/babou.rb

```diff
@@ -1,3 +1,5 @@
+require 'active_support/notifications'
```

Comment:
Let's change this single quoted string to double quotes.

I think we need to rewrite CI to use circle-ci too. Bummer.